File: backend/src/main/java/com/example/asplatform/common/enums/RepairStatus.java
package com.example.asplatform.common.enums;

public enum RepairStatus {
    PENDING,                 // 접수대기
    WAITING_FOR_REPAIR,      // 수리대기(접수+배정 완료)
    IN_PROGRESS,             // 수리중
    WAITING_FOR_PAYMENT,     // 결제대기(최종 견적 입력)
    WAITING_FOR_DELIVERY,    // 발송대기(결제 완료 후)
    COMPLETED,               // 처리완료(발송 완료)
    CANCELED                 // 취소
}

File: frontend/src/constants/repairStatus.js
const RepairStatusMap = {
    PENDING_APPROVAL: 1,
    WAITING_FOR_REPAIR: 2,
    IN_PROGRESS: 3,
    WAITING_FOR_PAYMENT: 4,
    WAITING_FOR_DELIVERY: 5,
    COMPLETED: 6,
    CANCELLED: 7,
};

const StepToStatusCode = {
  1: "PENDING_APPROVAL",
  2: "WAITING_FOR_REPAIR",
  3: "IN_PROGRESS",
  4: "WAITING_FOR_PAYMENT",
  5: "WAITING_FOR_DELIVERY",
  6: "COMPLETED",
  7: "CANCELLED",
};

const RepairStepLabels = [
    "접수 대기",
    "수리 대기",
    "수리 중",
    "결제 대기",
    "발송 대기",
    "발송 완료",
    "취소",
];

export { RepairStatusMap, StepToStatusCode, RepairStepLabels };